command/init.go

```diff
@@ -50,6 +51,15 @@ func (c *InitCommand) Run(args []string) int {
 		RecoveryPGPKeys:   recoveryPgpKeys,
 	}
 
+	switch len(rootTokenPgpKey) {
+	case 0:
+	case 1:
+		initRequest.RootTokenPGPKey = rootTokenPgpKey[0]
```

Comment:
Either here or in the Initialize() method (if CLI shouldn't do it), can we verify if the key supplied has proper headers?